catboost/python-package/catboost/core.py

```diff
@@ -5826,3 +5826,71 @@ def to_classifier(model):
     for attr in model.__dict__:
         setattr(classifier, attr, getattr(model, attr))
     return classifier
+
+
+from catboost.utils import read_cd
+
+
+class ColumnDescription:
+    column_type_synonyms_map = {
+        'Target': 'Label',
+        'DocId': 'SampleId',
+        'QueryId': 'GroupId'
+    }
+
+    def __init__(self, cd_file=None, column_count=None, data_file=None, canonize_column_types=False):
```

Comment:
it should be possible to create correct ColumnDescription passing `column_count` only